trello_cards.py

```diff
@@ -0,0 +1,72 @@
+import requests
```

Comment:
This file looks very unfinished. If you need help you should write a comment explaining what you've got stuck on, or you could book a 1 on 1 meeting with one of us to help solve any problems.

Reply:
file has been cleaned up.